gmso/core/element.py

```diff
@@ -118,17 +124,22 @@ def element_by_atomic_number(atomic_number):
         otherwise raise GMSOError
     """
     if isinstance(atomic_number, str):
-        atomic_number_trimmed = int(sub('[a-z -]', '', atomic_number.lower()).lstrip('0'))
-        msg = '''Letters and spaces are not considered when searching by element atomic number. \n
-                 {} became {}'.format(atomic_number, atomic_number_trimmed)'''
+        atomic_number_trimmed = int(
+            sub("[a-z -]", "", atomic_number.lower()).lstrip("0")
+        )
+        msg = """Letters and spaces are not considered when searching by element atomic number. \n
+                 {} became {}'.format(atomic_number, atomic_number_trimmed)"""
```

Comment:
same comment as above

Reply:
Fixed to use format strings from Python.